app/assets/stylesheets/components/data-portal/c-modal-download-all.scss

```diff
@@ -0,0 +1,54 @@
+.c-modal-download-all {
+  padding: 30px;
```

Comment:
I would put more padding to avoid long lines. If you take a look at the other modals, it's not unified but some of them have more space. It's just a suggestion.